### PR TITLE
Fixing duplicate SSS

### PR DESCRIPTION
**Pull request by jewzaam**
https://issues.redhat.com/browse/OSD-3032

There were 2 SSS with the same name, so the last one always overwrote the first.

### CODE REVIEW

File: hack/templates/template.yaml [hunk 1]
```diff
@@ -30,53 +30,4 @@ parameters:
...
-      kind: Subscription
-      metadata:
-        name: ${REPO_NAME}
-        namespace: openshift-${REPO_NAME}
-      spec:
-        channel: ${CHANNEL}
-        name: ${REPO_NAME}
-        source: ${REPO_NAME}-registry
-        sourceNamespace: openshift-${REPO_NAME}
+objects: []
```
Comment (cblecker) [inline hack/templates/template.yaml]: Is this right? It doesn't look right.

**Comment by jewzaam** [comment]
/hold

ci.ext doesn't have `jq`.. heart breaking!

File: hack/templates/template.yaml (hunk 1 above)
Comment (jewzaam) [reply hack/templates/template.yaml]: Script takes this and populates with child objects, which are SSS in our use case.  It's an array and we have nothing to start with, it is moved to separate yaml in deploy now (CatalogSource and Subscription).  To make sure python uses the right data type, `[]` so yes it's right.

**Comment by jewzaam** [comment]
Removed bits for the PR check tweaks, pushed to another branch and I'll hack on it later.

**Pull request merged**